TidepoolOnboarding/Assets.xcassets/HowTheAppWorks_10_1.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "HowTheAppWorks_10_1@2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```

Comment:
[nit] I'm still confused by all this `1x`, `2x`, and `3x`, given the contents for `HowTheAppWorks_10_1.imageset` above.

Reply:
[nit] I see `.png` are being added for `2x`. I thought images should be `.pdf`?

Reply:
I do wonder if using single scale is better for these assets.

Reply:
I was explicitly instructed by the design team to use PDFs for the all-vector graphics and PNGs for the screenshots.

I'm not sure what you are asking me to do here.

Do we all agree that the all-vector graphics should be PDF set as "Single Scale" in Xcode? I think that is the only thing that makes sense from both a scalability and disk size perspective.

Regarding the screenshot PNGs, are you suggesting to:
1. change them all to "Single Scale" in Xcode, or
2. change them all into PDFs set as "Single Scale" in Xcode, or
3. something else?

Please let me know. Thanks!

Reply:
@nhamming @ps2 I'd really like to resolve these concerns you both have so they won't come up in future onboarding work. Any clarification is greatly appreciated. Thanks!

Reply:
I remember a discuss within the devs to only use `.pdf`s and this is where my comment is coming from. Curious why design wants `.png` instead.